core-bundle/src/Search/Document.php

```diff
@@ -110,7 +110,7 @@ public function extractJsonLdScripts(string $context = '', string $type = ''): a
             ->filterXPath('descendant-or-self::script[@type = "application/ld+json"]')
             ->each(
                 static function (Crawler $node) {
-                    $data = json_decode($node->text(), true);
+                    $data = json_decode($node->text(null, true), true);
```

Comment:
I don't understand. If `true` was the default and will still be the default in the future, why do we explicitly have to set it?

Reply:
It is not technically necessary with the release of `terminal42/escargot` `1.0.5`. However, not setting the parameters will still result in 

```php
@trigger_error(sprintf('"%s()" will normalize whitespaces by default in Symfony 5.0, set the second "$normalizeWhitespace" argument to false to retrieve the non-normalized version of the text.', __METHOD__), \E_USER_DEPRECATED);
```

being triggered under certain circumstances.

Reply:
Then we should ignore the deprecation in these places as we do not want the non-normalized version of the text.

Reply:
If we do not want the non-normalized version we have to use `->text(null, true)` in Symfony 4.